discord_bot/chat_handler.py

```
@@ -542,18 +542,33 @@ async def on_message(message):
 
             if reply_to_message and reply_to_message.attachments:
                 for attachment in reply_to_message.attachments:
                     if attachment.content_type and attachment.content_type.startswith("image/"):
                         images.append(Image(url=attachment.url))
                         logger.info(f"[chatbot] Found reply image attachment: {attachment.url}")
+            
+            # Add reaction to acknowledge non-image attachments (user feedback)
+            if non_image_attachments:
+                try:
+                    await message.add_reaction('\U0001F4CE')  # 📎 paperclip emoji
+                    logger.info(f"[chatbot] Added paperclip reaction for {len(non_image_attachments)} non-image attachment(s)") 
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)